packages/nextjs/package.json

```diff
@@ -8,6 +8,7 @@
     "dev": "concurrently \"yarn next:dev\"",
     "next:dev": "next dev",
     "next:dev:mock": "NEXT_PUBLIC_API_MOCKING=enabled next dev",
+    "deploy-studio": "sanity deploy",
```

Comment:
Do we need this if the studio is embedded?

Reply:
@gksander Yeah, I wasn’t sure. I’ll drop it. Easy to add back if we want it later.